ApplicationLibCode/ProjectDataModel/Summary/RimSummaryPlot.cpp

```diff
@@ -1418,11 +1421,12 @@ void RimSummaryPlot::updateStackedCurveData()
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-void RimSummaryPlot::updateStackedCurveDataForAxis( RiuPlotAxis plotAxis )
+bool RimSummaryPlot::updateStackedCurveDataForAxis( RiuPlotAxis plotAxis )
 {
     std::map<RiaDefines::PhaseType, size_t> curvePhaseCount;
 
     auto stackedCurves = visibleStackedSummaryCurvesForAxis( plotAxis );
+    if ( stackedCurves.empty() ) return false;
```

Comment:
Move to top of method? No need to make the curvePhaseCount map if no visible curves.